tools/release/release-output/output-validations.ts

```diff
@@ -126,8 +128,47 @@ export function checkMaterialPackage(packagePath: string): string[] {
  */
 export function checkCdkPackage(packagePath: string): string[] {
   const prebuiltFiles = glob('*-prebuilt.css', {cwd: packagePath}).map(path => basename(path));
-
   return ['overlay', 'a11y', 'text-field']
-      .filter(name => !prebuiltFiles.includes(`${name}-prebuilt.css`))
-      .map(name => `Could not find the prebuilt ${name} styles.`);
+                       .filter(name => !prebuiltFiles.includes(`${name}-prebuilt.css`))
```

Comment:
nit: indentation got weird for some reason

Reply:
This is actually what clang-format did when I wrapped the expression for testing in an array.